Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM ubuntu:18.04 AS base1
```

Comment:
Is this the convention (to use `baseN`)? If not, then can we use more explicit naming?

Reply:
I'm not aware of any convention here, or even widespread use of multistage build files. I did this because it seemed pretty obvious, given the linear read of the file. It's not really clear to me different names are more clear. It's all a bit arcane docker junk